pkgs/top-level/splice.nix

```diff
@@ -65,7 +65,7 @@ let
   splicedPackages =
     if actuallySplice
     then splicer defaultBuildScope defaultRunScope
-    else pkgs // pkgs.xorg;
+    else import ./build-wrappers.nix pkgs // pkgs.xorg;
```

Comment:
I don't understand what you want to do, what invariant are you trying to maintain?

I just discovered that `pkgs.buildPackages` today, where can I read more about it.  I don't see where the attribute is being defined.